tox.ini

```diff
@@ -1,6 +1,22 @@
 [tox]
-envlist = py26, py27, py33, py34, py35, pypy
+envlist = py{27,33,34,35,py}-requests{,2.1.0,2.8.1,2.9.1},codestyle
 
 [testenv]
+deps =
+    pylint
```

Comment:
This way test dependencies are defined in the same file that defines other test environment options.
Both tests and package install no longer use requirements.txt